docs/root/configuration/upstream/cluster_manager/cluster_stats.rst

```diff
@@ -190,6 +190,11 @@ Circuit breakers statistics will be rooted at *cluster.<name>.circuit_breakers.<
   remaining_rq, Gauge, Number of remaining requests until the circuit breaker reaches its concurrency limit
   remaining_retries, Gauge, Number of remaining retries until the circuit breaker reaches its concurrency limit
 
+.. note::
+  Metrics starting with prefix "remaining_" are not generated by default. To track the number of resources remaining until a circuit breaker opens,
```

Comment:
hi @mashun4ek it seems like sphinx is interpreting `"remaining_"` as an anchor - could you make this an inline literal with double backticks - ie ` ``remaining_`` `